libraries/AP_HAL_ChibiOS/hwdef/MambaF405-2022/defaults.parm

```diff
@@ -2,4 +2,15 @@
 SERVO7_FUNCTION 120
 NTF_LED_TYPES 257
 # Default VTX power to on
-RELAY_DEFAULT 1
\ No newline at end of file
+RELAY_DEFAULT 1
+# UART1 is RX input
+SERIAL1_PROTOCOL 23
+#  UART2 is GPS
+SERIAL2_PROTOCOL 5
+# UART3 is DJI Goggle (MSP)
+SERIAL3_PROTOCOL 32
```

Comment:
We shouldn't do this - both DJI and regular VTX are listed as UART3 - we should just leave it up to the user to decide. Also UART5 is listed as F.Port - we should think whether we want to include that

Reply:
sorry, I disagree..the way you have it defined now, its a GPS...the most logical is to assign it to a video setup and since there is an AIR connector on the board...that's most logical....non DJI user of CAM control would need to change it, but DJI users would, if its the other way...I'd opt for the connector on the board...ie DJI
and there is NO UART5 pinned out....
there are only 3 Diatone F405 bds being produced....the F405MK2 which uses the -I2US definition....and the two F405 minis. using this definition (-2022)...none of which have UART5